template/cordova/lib/target-list.js

```diff
@@ -17,20 +17,21 @@
        under the License.
 */
 
-var devices = require('./package'),
+var deploy  = require('./deployment'),
     args = process.argv.slice(2);
 
 // help/usage function
 function help() {
     console.log('');
-    console.log('Usage: node target-list.js  [ --emulators | --devices | --started_emulators | --all ]');
+    console.log('Usage: node target-list.js [--win10] [ --emulators | --devices | --started_emulators | --all ]');
+    console.log('    --win10             : Chooses to list Windows 10 devices (Windows 8.1 is default).');
     console.log('    --emulators         : List the possible target emulators availible.');
-    console.log('    --devices           : List the possible target devices availible. *NOT IMPLEMENTED YET*');
+    console.log('    --devices           : List the possible target devices availible.');
     console.log('    --started_emulators : List any started emulators availible. *NOT IMPLEMENTED YET*');
     console.log('    --all               : List all available devices');
     console.log('examples:');
     console.log('    node target-list.js --emulators');
-    console.log('    node target-list.js --devices');
+    console.log('    node target-list.js --win10 --devices');
```

Comment:
does the cordova run --list command need an update? It will be nice to list both set of devices/emulators for Win10 & Win8.1 rather than have to specify an argument to pick and chose for listing purposes.